### PR TITLE
Adding Just the Gallery

### PR DESCRIPTION
**Pull request by nabobalis**
Just the gallery. Nothing else.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -70,10 +73,11 @@ jobs:
           fingerprints: "b1:c8:b0:94:a8:ec:67:16:2b:0f:18:a9:49:a6:b1:db"
       - run: ssh-keyscan github.com >> ~/.ssh/known_hosts
       - run: git config --global user.email "ndcube@circleci" && git config --global user.name "ndcube Circle CI"
-      - run: git clone git@github.com:sunpy/sunpy-figure-tests.git --depth 1 -b ndcube-${CIRCLE_BRANCH} ~/sunpy-figure-tests/
+      - run: git clone git@github.com:sunpy/sunpy-figure-tests.git --depth 1 -b sunpy-${CIRCLE_BRANCH} ~/sunpy-figure-tests/
```
Comment (Cadair) [inline .circleci/config.yml]:
```suggestion
      - run: git clone git@github.com:sunpy/sunpy-figure-tests.git --depth 1 -b ndcube-${CIRCLE_BRANCH} ~/sunpy-figure-tests/
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -82,30 +86,31 @@ jobs:
           git pull
           git status
           git add .
-          git commit -m "Update reference figures from ndcube ${CIRCLE_BRANCH}" || echo "No changes to reference images to deploy"
+          git commit -m "Update reference figures from ${CIRCLE_BRANCH}" || echo "No changes to reference images to deploy"
```
Comment (Cadair) [inline .circleci/config.yml]:
```suggestion
          git commit -m "Update reference figures from ndcube ${CIRCLE_BRANCH}" || echo "No changes to reference images to deploy"
```

File: tox.ini [hunk 3]
```diff
@@ -13,18 +13,10 @@ isolated_build = true
 pypi_filter = https://raw.githubusercontent.com/sunpy/sunpy/main/.test_package_pins.txt
 # We use bash in some of our environments so we have to whitelist it.
 whitelist_externals=
-    /bin/bash
+    /bin/bashgit@github.com:sunpy/ndcube.git
```
Comment (Cadair) [inline tox.ini:16]: err?

Comment (nabobalis) [reply tox.ini]: Copy and paste went wrong.

File: tox.ini [hunk 4]
```diff
@@ -34,50 +26,39 @@ setenv =
...
-
-# The following indicates which extras_require from setup.cfg will be installed
-# We use test and all so we don't install docs when we don't need it (as
-# opposed to dev).
+    figure: matplotlib==3.5.2
+    figure-!devdeps: astropy==5.1.0
+    figure-!devdeps: mpl-animators==1.0.0
+    # TODO
+    # https://github.com/aio-libs/aiohttp/issues/6799
+    figure-devdeps: aiohttp<4
```
Comment (nabobalis) [inline tox.ini:61]: LOL

Comment (nabobalis) [reply tox.ini:61]: To be moved upstream later on.

**Comment by nabobalis** [comment]
Backport?

File: ndcube/utils/wcs.py [hunk 5]
```diff
@@ -454,7 +454,7 @@ def get_low_level_wcs(wcs, name='wcs'):
     elif isinstance(wcs, BaseLowLevelWCS):
         return wcs
     else:
-        raise(f'{name} must implement either BaseHighLevelWCS or BaseLowLevelWCS')
+        raise (f'{name} must implement either BaseHighLevelWCS or BaseLowLevelWCS')
```
Comment (Cadair) [inline ndcube/utils/wcs.py:457]:
```suggestion
        raise f'{name} must implement either BaseHighLevelWCS or BaseLowLevelWCS'
```

File: tox.ini (hunk 4 above)
Comment (nabobalis) [inline tox.ini:61]:
```suggestion
```

**Pull request merged**